src/mlst_download_functions.jl

```diff
@@ -181,50 +181,48 @@ function download_enterobase_scheme(scheme, s_type, output_dir, overwrite=false)
   return loci_files
 end
 
-function download_cgmlst_scheme(target_id, output_dir, overwrite=false)
+function download_cgmlst_scheme(target_id, output_dir)
```

Comment:
the `overwrite` parameter wasn't being used, so I've removed it. I do overwrite any existing scheme files using `unzip -o`